server/routes/api/reports.js

```diff
@@ -0,0 +1,35 @@
+import express from 'express';
+import Authenticate from '../../middlewares/Authenticate';
+import ReportsController from '../../controllers/ReportsController';
+import ReportValidation from '../../middlewares/validations/ReportValidation';
+import Queryvalidation from '../../middlewares/validations/QueryValidation';
```

Comment:
`Queryvalidation` should be `QueryValidation`

Reply:
Thank you, Fixed.